### PR TITLE
Super GameBoy 2 variant cores & Other stuff

### PR DESCRIPTION
**Pull request by hallem**
- Added support for the Super GameBoy 2 & Super GameBoy 2: Vaporwave Edition variant cores
- Added support to change the aspect ratio on the Super GameBoy cores to 8:7 and back to 4:3
- Added missing event bindings to output more information to the console since the PocketCoreUpdater isn't being used for the extras.
- Added descriptions for the Pocket Extras that are displayed before the installation happens with a user confirmation
- Added a setting to suppress the above menu descriptions
- Fixed an issue with the newer command line verbs that threw an error when a path wasn't specified
- Added support for list and info options on pocket-extras verb to display detailed information about the options

### CODE REVIEW

File: src/partials/Program.HelpText.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ internal partial class Program
     -p, --path                Absolute path to install location
     -n, --name                The name of the extra to install. Required
                                  Values: DonkeyKong, RadarScope, jtbubl, jtcps1, jtcps15,
-                                         jtcps2, jtpang, toaplan2_c, jtc16_c
+                                         jtcps2, jtpang, toaplan2_c, jtc16_c, sgb2, sgbvw
```
Comment (mattpannella) [inline src/partials/Program.HelpText.cs:56]: i think a good future improvement here would be we can add a --list and --info option. so we don't need to hardcode the list here, but let the user display it dynamically, and then they can also get the description via --info

Comment (hallem) [reply src/partials/Program.HelpText.cs:56]: So --list would list out the values and maybe the descriptions?
And --info would do what?

Comment (mattpannella) [reply src/partials/Program.HelpText.cs:56]: yeah didn't think about the required name. don't have to worry about it right now

Comment (hallem) [reply src/partials/Program.HelpText.cs:56]: Turns out the required attribute doesn't display an error message when it's missing, just terminates. So not using it is actually better. Sadly.

Comment (hallem) [reply src/partials/Program.HelpText.cs]: Got both list and info added as well as a fix for the newer cli commands erroring out when a path isn't specified.

**Pull request merged**